OneSignalSDK/onesignal/core/src/main/java/com/onesignal/UserJwtInvalidatedEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.onesignal
+
+/** TODO: jwt documentation
+ * The event passed into [IUserJwtInvalidatedListener.onUserJwtInvalidated], it provides access
+ * to the external ID whose JWT has just been invalidated.
+ *
+ */
+class UserJwtInvalidatedEvent(
+        val external_id: String
```

Comment:
Remove underscore, rename to `externalId`.

This was a mistake in the spec, and just corrected it there.